src/Components/Referrer/referrerDataInfo.ts

```diff
@@ -0,0 +1,70 @@
+import MFBCOLogo from '../../Assets/Logos/mfb_colorado_logo.png';
+import MFBNCLogo from '../../Assets/Logos/mfb_nc_logo.png';
+import BIAMFBLogo from '../../Assets/biamfbcombinedlogo.png';
+import JHSAMFBLogo from '../../Assets/JeffcoAssets/jeffcobrand.png';
+import VELogo from '../../Assets/VillageExchange/villageExchangeLogo.png';
+import CCHMFBLogo from '../../Assets/ColoradoCoalitionHomeless/cchcobrand.png';
+import LGSLogo from '../../Assets/LetsGetSet/lgsLogo.png';
+import GACLogo from '../../Assets/GetAheadColorado/gaclogo.png';
+import FIRCLogo from '../../Assets/FircCobrand/FIRCLogo.png';
+import COMFBLogo from '../../Assets/colorado-mfb-logo.png';
+import DHSMFBLogo from '../../Assets/DenverHumanServices/denverHSLogo.png';
+
+export const coLogoSource = {
+  default: MFBCOLogo,
+  bia: BIAMFBLogo,
+  jeffcoHS: JHSAMFBLogo,
+  jeffcoHSCM: JHSAMFBLogo,
+  villageExchange: VELogo,
+  cch: CCHMFBLogo,
+  lgs: LGSLogo,
+  gac: GACLogo,
+  fircsummitresourcecenter: FIRCLogo,
+  coBenefits: COMFBLogo,
+  dhs: DHSMFBLogo,
+};
```

Comment:
This can be added to the `referrerData` in the config. You just need to map a key to the logo like we do with the icons.